src/Raven.Client/ServerWide/Operations/SetDatabaseDynamicDistribution.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class SetDatabaseDynamicDistribution : IServerOperation<bool>
+    {
+        private bool _allowDynamicDistribution;
+        private string _databaseName;
+
+        public SetDatabaseDynamicDistribution(string databaseName, bool allowDynamicDistribution)
+        {
+            _allowDynamicDistribution = allowDynamicDistribution;
+            _databaseName = databaseName;
```

Comment:
Let's assert if this is not null or empty

Reply:
what is null?

Reply:
databaseName, string